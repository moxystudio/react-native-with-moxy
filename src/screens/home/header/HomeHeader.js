import React, { useMemo } from 'react';
import { View } from 'react-native';
import { Logo } from '../../../shared/media/icons';

import buildStyles from './styles';

const HomeHeader = () => {
    const styles = useMemo(() => buildStyles(), []);

    return (
        <View style={ styles.header }>
            <Logo width="120" height="50" />
        </View>
    );
};

export default HomeHeader;
