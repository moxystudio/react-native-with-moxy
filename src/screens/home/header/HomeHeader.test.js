import React from 'react';
import { mount } from 'enzyme';
import HomeHeader from '.';

it('should render correctly', () => {
    expect(mount(<HomeHeader />)).toMatchSnapshot();
});
