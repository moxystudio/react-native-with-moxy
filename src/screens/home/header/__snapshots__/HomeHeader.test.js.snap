// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<HomeHeader>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <logo.svg
        height="50"
        width="120"
      />
    </View>
  </View>
</HomeHeader>
`;
